lib/utils/matchUrl.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var { MatchPattern } = require("sdk/util/match-pattern");
+
+/**
+ * Check if URI is in whitelist
+ * @param include object {0: "*", ...}
+ * @param uri to test
+ * @return testResult true or false
+ */
+exports.isUriIncluded = function(include, uri) {
```

Comment:
OK, I've renamed it to includePatterns. (It is an array-like object of include URIs)